.gitignore

```diff
@@ -140,3 +140,11 @@ Round 1 Reporting - TF_aggregated_data_23.09.2022.xlsx
 
 # output files
 data_store_ERD.png
+ROUND_1_DATA_INGESTION.xlsx
+Data Model v3.7.xlsx
+.~lock.Round 1 Reporting - TF_aggregated_data_23.09.2022.xlsx#
+.~lock.ROUND_1_DATA_INGESTION.xlsx#
+~$Post_transform_EXAMPLE_round_one_data.xlsx
+
+# environments
+myenv/
```

Comment:
Can we revert this stuff? You can put it in your local gitignore global if you want.